### DISCUSSION TITLE
<GitHub Pages startup Failure>

### DISCUSSION THREAD

**Discussion by shiranD** [community/community#87690] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I have been working on a website template to create a personal website. As part of adjusting it, I both modified files as well as deleted some. I was able to deploy the website, but at some point, the updates I was providing to master branch stopped being reflected in gh-pages branch ([my website](https://github.com/shiranD/shirand.github.io)). One such update was deleting files* I don't use in my website such as _pages/repositories.md (or _pages/teaching.md). Ever since I did that, it only showed a "startup failure" [here](https://github.com/shiranD/shirand.github.io/actions) for all other actions (except for one). One common comment I see

"[Error: .github#L1](https://github.com/shiranD/shirand.github.io/commit/e82826d65820327f035eb9cf339da63cb643baba#annotation_16983117073)
actions/checkout@v3, docker/setup-qemu-action@v2, docker/setup-buildx-action@v2, docker/login-action@v2, and docker/build-push-action@v4 are not allowed to be used in shiranD/shirand.github.io. Actions in this workflow must be: within a repository owned by shiranD."

I am not sure what the meaning of the comment, and will appreciate any help on this to make it be deployed again, and possibly how to avoid it in future (I also mistakingly deleted the specific workflow in Actions showing the deletion attempt of _pages/repositories.md, but other ones failed the same).

*which was done through the GitHub api website, and not my local machine

**Comment by 0foxes** [comment]
The workflow belongs to your repository, but the action itself belongs to `docker`. You should go to repository settings (under Actions) to enable external actions.

**Reply by 0foxes** [reply, marked as answer]
![image](https://github.com/community/community/assets/31290848/174410e3-f807-4276-8ad7-f1b1a03a081a)
(the first option)

**Reply by shiranD** [reply]
I will do that! Thanks, indeed around that point I changed it to allow my user actions and reusable workflows. Now that I had changed it, I am not sure how to move forward? How can the master branch operations be reflected somehow in the gh-pages branch, or I should manually do something else. For instance, I already deleted a file in the master branch, so what can I do given the new permissions I set today, to have the deleted file be reflected in gh-pages. Ideally, have all changes be reflected, but also go one by one if I have to.
Thanks!

**Reply by shiranD** [reply]
I actually figured out the next step which is to rerun the workflow. After doing that, my website now reflects those updates. Thanks again!

**Reply by 0dayScorpio** [reply]
Hey, can you help me with something?
When I load some GITHub page on my Linux, it doesn't show the code/information about the page. For example you go to "remcos" download and for me it just stays blank, does't show any text at all.
Thanks!

**Comment by 0foxes** [comment]
Thanks, are you able to mark my answer as the solution?
